todoapp/app/src/main/AndroidManifest.xml

```diff
@@ -36,7 +36,7 @@
         <activity android:name="com.example.android.architecture.blueprints.todoapp.taskdetail.TaskDetailActivity" />
         <activity android:name="com.example.android.architecture.blueprints.todoapp.addedittask.AddEditTaskActivity" />
         <activity
-            android:name="com.example.android.architecture.blueprints.todoapp.statistics.StatisticsActivity"
+            android:name="com.example.android.architecture.blueprints.todoapp.statistics.ui.StatisticsActivity"
```

Comment:
why did you change the package?

Reply:
It made more sense to use `ui` than `view`. I'll change it back

Reply:
Ah but the Activity wasn't part of the view package. Nor is the fragment.

Reply:
right, it wasn't. I moved everything related to the UI (activity, fragment, controller) to the `ui` package so it would be separated from the mobius logic